nbgrader/server_extensions/formgrader/static/js/manage_assignments.js

```diff
@@ -188,6 +192,27 @@ var AssignmentUI = Backbone.View.extend({
                 .attr("href", base_url + "/formgrader/manage_submissions/" + this.model.get("name"))
                 .text(num_submissions));
         }
+
+        // generate feedback
+        if (num_submissions > 0) {
+            this.$generate_feedback.append($("<a/>")
+		.attr("href", "#")
+                .click(_.bind(this.generate_feedback, this))
+		.append($("<span/>")
+		   .addClass("glyphicon glyphicon-star-empty")
+                   .attr("aria-hidden", "true")));
+        }
+
+        //  feedback
+        if (num_submissions > 0) {
```

Comment:
This will be based on any available to release. Should we keep track of those that haven't been sent, so one could send incrementally, as you have them graded, or allow for re-sending if never received?